.github/workflows/coc-camel-dev.yaml

```diff
@@ -19,7 +19,6 @@ jobs:
     - uses: actions/setup-node@v3
       with:
             node-version: 16
-            cache: 'yarn'
```

Comment:
why removing the cache?

Reply:
> why removing the cache?

I have removed yarn.lock in this PR. When the workflow is running is without yarn.lock it was failing with below errors . To fix the errors i have removed the cache line
`Error: Dependencies lock file is not found in /home/runner/work/coc-camel/coc-camel. Supported file patterns: yarn.lock`

Reply:
I have included yarn.lock file in the recent PR that was merged and when I do the PlugInstall with having yarn.lock file is producing the following errors. 
 ```                                                                            |
x Post-update hook for coc-camel ... Exit status: 1                                                |" Shorthand notation; fetches https://github.com/neoclide/coc.nvim
- coc-camel: Receiving objects: 100% (18/18), 143.88 KiB | 975.00 KiB/s, done.                     |Plug 'neoclide/coc.nvim', {'branch': 'release'}
- coc.nvim: Already installed  
```   
Errors were not there before the lock file so decided to remove the lock file from the repo to fix the errors.

Reply:
fine to remove the lock for this time. Be aware that providing a .lock is a good practice of Software Maintenance. it allows having reproducible build. Without it, if we want need to rebuild the version, or something based on this version later, different dependencies version might be picked up. Which then can cause troubles if not compatible.

Reply:
Thank you for explaining it. I believe I found where it was causing the errors. I see errors below when I run yarn install.
```
yarn install v1.22.19
[1/5] Validating package.json...
warning coc-camel@0.0.0-development: The engine "coc" appears to be invalid.
[2/5] Resolving packages...
error Your lockfile needs to be updated, but yarn was run with `--frozen-lockfile`.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.

shell returned 1
```
Now I have updated the new yarn.lock file